tests/tests/worm-xattr-unit.sh

```diff
@@ -0,0 +1,49 @@
+t_require_commands touch rm setfattr
+
+echo "== test creation"
+touch "$T_D0/file-1"
+SECS=$(date '+%s')
+NSECS=$(date '+%N')
+DELAY=10
+EXP=$(( $SECS + $DELAY ))
+setfattr -n scoutfs.worm.v1_expiration -v $EXP.$NSECS "$T_D0/file-1"
```

Comment:
If we're requiring .hide., it'll be added to all the calls except a test that now requires it :).